src/cider/examples/CiderRuntimeModuleExample.cpp

```diff
@@ -69,8 +63,8 @@ int main(int argc, char** argv) {
 
   auto ciderCompileModule = CiderCompileModule::Make(allocator);
   auto result = ciderCompileModule->compile({add_expr},
```

Comment:
Add a TODO that later we will move to new evaluator API.

Reply:
These files will be removed soon. New tests will surely be added during new API development.